code/src/album.css

```diff
@@ -0,0 +1,122 @@
+
+:root {
+  --primary-headings-font-family: "Helvetica", sans serif;
+  --secondary-headings-font-family: "Helvetica", sans serif;
+  --paragraph-font-family: "Helvetica", sans serif;
+}
+
+/*ALBUM*/
+
+.album-name {
+  font-family: var(--secondary-headings-font-family);
+  font-size: 14px;
+  color: #fff;
+  text-decoration: none;
+}
+
+.album {
+  margin-bottom: 1.5rem;
+}
+
+.album-name:hover {
+  text-decoration: underline;
+}
+
+.album-cover {
+  width: 100%;
+  height: 100%;
+}
+
+/*album-cover gets opacity at hover on its parent element cover-and-icons*/
+
+.cover-and-icons:hover .album-cover {
+  opacity: 0.5;
+  cursor: pointer;
+}
+
+.album-container {
+  place-items: center;
+}
+
+/*BUTTONS*/ 
+
+.cover-and-icons  {
+    position: relative;
+}
+
+/*img-overlay is displayed in block at hover on its parent element cover-and-icons. This makes the icons visible at hover on the album*/
+
+.cover-and-icons:hover .img-overlay {
+    display: block;
+}
+
+.img-overlay {
+    justify-content: center;
+    align-items: center;
+    display: none;
+}
+
+.icons-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 1;
```

Comment:
We used transition to do the same but we're curious to hear more about your solution at next weeks Team sessions :) 
/Ylva & Emma